cmd/bb_storage/main.go

```diff
@@ -40,6 +52,32 @@ func main() {
 		log.Fatal(http.ListenAndServe(*webListenAddress, nil))
 	}()
 
+	pe, err := prometheus_exporter.NewExporter(prometheus_exporter.Options{
+		Namespace: "bb_storage",
+		Registry: prometheus.DefaultRegisterer.(*prometheus.Registry),
+	})
+	if err != nil {
+		log.Fatalf("Failed to create the Prometheus stats exporter: %v", err)
+	}
+	view.RegisterExporter(pe)
+	if err := view.Register(ocgrpc.DefaultServerViews...); err != nil {
+		log.Fatalf("Failed to register ocgrpc server views: %v", err)
+	}
+	zpages.Handle(nil, "/debug")
+	je, err := jaeger.NewExporter(jaeger.Options{
+		AgentEndpoint:          *agentEndpointURI,
+		CollectorEndpoint:      *collectorEndpointURI,
+		ServiceName:            *serviceName,
```

Comment:
See [documentation](https://godoc.org/contrib.go.opencensus.io/exporter/jaeger#Options): ServiceName is deprecated and [Process](https://godoc.org/contrib.go.opencensus.io/exporter/jaeger#Process) should be used instead.
It might be worth noting that basic auth is also then not supported. My guess is that it isn't used that much, but maybe this should be either documented or made configurable as well?

Reply:
thank you for the  great catch